ocs_ci/ocs/must_gather/const_must_gather.py

```diff
@@ -17,7 +17,6 @@
 
 
 GATHER_COMMANDS_CEPH = [
-    "ceph-volume_raw_list",
```

Comment:
@OdedViner Why did we remove this command from ceph commands list?
On 4.9 must-gather command collects "ceph-volume_raw_list" output

Reply:
Some tests failed on CI.. I will ask dev team. https://ocs4-jenkins-csb-ocsqe.apps.ocp4.prod.psi.redhat.com/job/qe-deploy-ocs-cluster-prod/2293/testReport/tests.manage.z_cluster.test_must_gather/TestMustGather/test_must_gather_CEPH_/